src/Microsoft.IO.Redist/src/System/Runtime/InteropServices/SuppressGCTransitionAttribute.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Internal copy of SuppressGCTransitionAttribute used only to enable files
+// that use this to compile.  On any platforms that support this attribute,
+// the real one exposed from CoreLib should be used.
+
+namespace System.Runtime.InteropServices
+{
+    [AttributeUsage(AttributeTargets.Method, Inherited = false)]
+    internal sealed class SuppressGCTransitionAttribute : Attribute
+    {
+    }
+}
```

Comment:
@AaronRobinsonMSFT, @JeremyKuhne, just FYI.  This OOB netfx assembly is using files that use this new attribute, so I've added this non-functional internal copy of it into Microsoft.IO.Redist so that everything will compile.  The attribute should simply be a nop.